js/_api.js

```diff
@@ -20,21 +20,27 @@ export function call(uri, method, payload) {
 		credentials: 'same-origin',
 		method: method,
 		headers: {
-			Accept: 'application/json',
-			'Content-Type': 'application/json'
+			Accept: 'application/json'
 		}
 	};
 
-	if(payload) {
-		var data = new FormData();
-		data.append("json", JSON.stringify(payload));
-		options.body = JSON.stringify(payload);
+	if(method.toLowerCase() === "post") {
+		options.body = new FormData();
+		options.body.append(window.api_auth.name, window.api_auth.value);
+
+		var data = payload || {};
+		Object.keys(data).forEach(function(key) {
+			options.body.append(key, payload[key]);
+		});
 	}
 
 	return fetch(uri, options)
```

Comment:
You are making history. Well, I'd say at minimum the api helpers shouldn't be colocated with the API calls itself. We can standardise on an _api.js per module though for defining the actual permitted calls?